account/models.py

```diff
@@ -0,0 +1,60 @@
+from django.db import models
+from product.models import *
+
+
+class User(models.Model):
+	first_name	= models.CharField(max_length = 100)
+	last_name	= models.CharField(max_length = 100)
+	email		= models.CharField(max_length = 200)
+	password	= models.CharField(max_length = 100)
```

Comment:
```suggestion
	password	= models.CharField(max_length = 300)
```
좀더 여유있게 300으로 늘려주세요.